app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Here is the list of users</h1>
\ No newline at end of file
+<div class="wrapper">
+  <ul class="user-list">
+    <li class="user-list-item">
+      <img src="<%= asset_path( 'Placeholder.svg.png' ) %>" alt="user">
+      <div>
+        <h2><%= @user.name %></h2>
+        <p class="float-right">Number of posts: <span><%= @user.posts_counter %></span>
+      </div>
+    </li>
+  </ul>
+  <div class="bio border">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+   <ul class="post-list list-type-none">
+    <% @user.posts.each do |post| %>
+      <li class="post-list-item">
+        <div class="border post">
+          <h3><%= post.title %></h3>
+          <p><%= post.text %></p>
+          <ul class="comments-likes">
+            <li>Comments: <%= post.comments_counter %></li>
+            <li>Likes: <%= post.likes_counter %></li>
+          </ul>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <div class="action_btn">
+    <%= link_to 'See All Posts' %>
+  </div>
```

Comment:
- [ ] As required for the project, kindly make sure when the user clicks the `See all Posts` link/button, it opens user's post index(i.e. `app/views/posts/index.html.erb` page) as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)